lambdatimes/src/components/Content/Tabs.js

```diff
@@ -1,16 +1,34 @@
-import React from 'react';
-import Tab from './Tab';
+import React from "react";
+import Tab from "./Tab";
+import PropTypes from "prop-types"
+
 const Tabs = props => {
   return (
     <div className="tabs">
       <div className="topics">
         <span className="title">TRENDING TOPICS:</span>
         {/* map over the tabs provided on your props, create a new Tab component for each one.
-            give the tab component a `selectTabHandler`, the `selectedTab`, and the `tab` itself as props*/}
+            give the tab component a `selectTabHandler`, the `selectedTab`, and the `tab` itself as props*/
+          props.tabs.map(tab => {
+            return (
+              <Tab
+                key={tab}
+                tab={tab}
+                selectedTab={props.selected}
```

Comment:
`props.selected` is undefined

Reply:
Thanks Samar. This has now been resolved. The console is now showing the correct data in the selectedTab.